queries.sql

```diff
@@ -109,4 +109,51 @@ SELECT
 species, AVG(escape_attempts)
 FROM animals
 WHERE date_of_birth BETWEEN '01-01-1990' AND '01-01-2000'
-GROUP BY species
\ No newline at end of file
+GROUP BY species
+
+--------------------------------------------------------------------------------------------------------------------------
+
+-- What animals belong to Melody Pond?
+
+SELECT * FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE animals.owner_id = (SELECT id from owners WHERE full_name='Jennifer Orwell');
```

Comment:
- [ ] Please correct the full name here, I believe it's 'Melody Pond'

Reply:
Thank you , I did Fix the name